### PR TITLE
Add credit-card and conversion-tag icons

### PR DESCRIPTION
**Pull request by denamwangi**
This adds two new icons to our set of icons: credit-card and conversion-tag. I pulled them from PDS directly but I'm not sure if they're exported to match gestalt standards- please let me know if any adjustments are needed.

cc_conversion_tag_icons

## TODO

- [x] Documentation
- [ ] Tests
- [ ] Experimental evidence (required for Masonry changes)
- [ ] Accessibility checkup

### CODE REVIEW

**Comment by christianvuerings** [comment]
@denamwangi could you make sure these icons go through PDS? Looks like they haven't been optimized yet:

image

**Comment by denamwangi** [comment]
Hi @christianvuerings! Thanks for the feedback- what's the general rule of thumb/acceptable range for what we're looking to see for that output? 

We got this from the PDS export tool which is WIP and working toward alignment with gestalt. Someone (PJ!) will be in touch directly to discuss with you and also to update these.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 ### Minor
 
+- Icon: Add credit-card and conversion-tag icons (#715)
```
Comment (denamwangi) [inline CHANGELOG.md]: will update this

**Comment by christianvuerings** [comment]
@denamwangi For now, feel free to run them through `svgo` manually yourself. I'm working with PJ to figure out a better solution moving forward

**Pull request merged**